src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -0,0 +1,27 @@
+package juja.sqlcmd;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
+    private static final String POSTGRESQL_DRIVER = "org.postgresql.Driver";
+    private Connection connection;
+
+    public boolean connect(String databaseName, String user, String password) {
+        try {
```

Comment:
I think you can use one try and few catch blocks

Reply:
Thanks, you are right, i have changed it already.